<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-28T11:49:01 AEST" name="Suite" finished-at="2024-08-28T11:49:08 AEST" duration-ms="7430">
    <groups>
    </groups>
    <test started-at="2024-08-28T11:49:01 AEST" name="Test" finished-at="2024-08-28T11:49:08 AEST" duration-ms="7430">
      <class name="com.ots.testcases.LoginTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.ots.testcases.LoginTest@524f3b3a]" started-at="2024-08-28T11:49:02 AEST" name="setUp" finished-at="2024-08-28T11:49:04 AEST" duration-ms="2207" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validLogin(java.lang.String,java.lang.String)[pri:0, instance:com.ots.testcases.LoginTest@524f3b3a]" started-at="2024-08-28T11:49:04 AEST" name="validLogin" data-provider="LoginTestData" finished-at="2024-08-28T11:49:07 AEST" duration-ms="2779" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@email.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin@123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ots.testcases.LoginTest@524f3b3a]" started-at="2024-08-28T11:49:07 AEST" name="tearDown" finished-at="2024-08-28T11:49:07 AEST" duration-ms="164" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.ots.testcases.LoginTest@524f3b3a]" started-at="2024-08-28T11:49:07 AEST" name="setUp" finished-at="2024-08-28T11:49:08 AEST" duration-ms="1208" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyNewUserLink()[pri:0, instance:com.ots.testcases.LoginTest@524f3b3a]" started-at="2024-08-28T11:49:08 AEST" name="verifyNewUserLink" finished-at="2024-08-28T11:49:08 AEST" duration-ms="51" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[New User Creation link is not present expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: New User Creation link is not present expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at com.ots.testcases.LoginTest.verifyNewUserLink(LoginTest.java:40)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyNewUserLink -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ots.testcases.LoginTest@524f3b3a]" started-at="2024-08-28T11:49:08 AEST" name="tearDown" finished-at="2024-08-28T11:49:08 AEST" duration-ms="90" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ots.testcases.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
